diffkemp/simpll/DifferentialFunctionComparator.cpp

```diff
@@ -1431,3 +1436,20 @@ void DifferentialFunctionComparator::findDifference(
         }
     }
 }
+
+bool DifferentialFunctionComparator::equal(const Instruction *InstL,
+                                           const Instruction *InstR) {
+    auto L = sn_mapL.find(InstL);
+    auto R = sn_mapR.find(InstR);
+
+    if (L == sn_mapL.end() || R == sn_mapR.end())
+        // Values are not in maps
+        return false;
+
+    if (std::make_pair(InstL, InstR) == DifferingInstructions)
+        // Comparing instructions with the same serial number are the differing
+        // ones
```

Comment:
Better comment: the instructions are the last instructions that were compared as non-equal.